source/agora/node/FullNode.d

```diff
@@ -466,4 +486,65 @@ public class FullNode : API
     protected void onAcceptedTransaction () @safe
     {
     }
+
+    /***************************************************************************
+
+        Push the block data to the `block_handlers` target server list
+        set in config.
+
+        Convert block data to JSON serialization and send it POST using Rest.
+
+        Params:
+            block = externalized block data
+
+    ***************************************************************************/
+
+    public void pushBlock (const ref Block block) @safe
```

Comment:
Why is this public?

And why is it never called?

Reply:
I think it will be called outside the FullNode class.

Reply:
I'm preparing for the next PR.

Reply:
> I think it will be called outside the FullNode class.

???

Reply:
I'm thinking about when to call.
BlockStorage.saveblock thinks it's the starting point for `pushBlock` delegate events.

Reply:
You want BlockStorage to be the trigger point that calls FullNode.pushBlock? You want to add FullNode as an instance to the BlockStorage class?